bundles/rust/init.moon

```diff
@@ -0,0 +1,21 @@
+-- Copyright 2012-2016 The Howl Developers
```

Comment:
Nitpicking, but as this bundle was created this year the copyright should say "Copyright 2016".